README.md

```diff
@@ -1,68 +1,8 @@
-# Conan libraries and code wishlist
+# Go to [Conan Center Index instead](https://github.com/conan-io/conan-center-index)
 
-This repository is a community effort used to track requests for code and packages that users would like to have available
-in conan. This Readme might contain a **summary** of the requests and status, as well as recent news. But the
-main flow of information is via github issues, please read the following guide:
+This repository tracked user wishes for new Conan recipes prior to the start of the [Conan Center Index](https://github.com/conan-io/conan-center-index).
 
+Please go there now. If a Conan package does not exist in the Index yet, please search the issues first, if someone else already made a request for the library or tool you would like to see. If such a request does not exist, feel free to create one.
```

Comment:
```suggestion
- If a Conan package does not exist in the Index yet, please search the issues first;
- If someone else already made a request for the library or tool you would like to see;
- If such a request does not exist, feel free to [create](https://github.com/conan-io/conan-center-index/issues/new?labels=library+request&template=package_request.md&title=%5Brequest%5D+%3CLIBRARY-NAME%3E%2F%3CLIBRARY-VERSION%3E) one.
```

Reply:
Actually, that semicolon makes no sense there 🤔

Reply:
sorry! it's Portuguese grammar :sweat_smile: It was automatic